ra-tls/ra-tls-server/lib/Makefile

```diff
@@ -9,6 +9,11 @@ WOLFSSL_ROOT ?= $(shell readlink -f $(TOPDIR)/wolfssl)
 SGX_WOLFSSL_LIB ?= $(shell readlink -f $(WOLFSSL_ROOT)/IDE/LINUX-SGX)
 SGX_RA_TLS_ROOT ?= $(shell readlink -f $(TOPDIR)/sgx-ra-tls)
 
+ifdef ECDSA
+SGX_DCAP ?= SGXDataCenterAttestationPrimitives
+SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/ql/inc -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include -I../build/include/  -I/usr/x86_64-linux-gnu/include/ -I$(SGX_DCAP)/QuoteVerification/QVL/Src/AttestationLibrary/include/ -I/usr/include/x86_64-linux-gnu/ -I$(SGX_DCAP)/QuoteGeneration/qpl/inc
```

Comment:
Plz split this single line into multi-lines. And plz remove the unnecessary trailing "/".